examples/va/hci/control-plane.md

```diff
@@ -38,7 +38,7 @@ oc wait nncp -l osp/nncm-config-type=standard --for jsonpath='{.status.condition
 
 Generate the control-plane and networking CRs.
 ```
-kustomize build > control-plane.yaml
+kustomize build control-plane > control-plane.yaml
```

Comment:
The NetConfig generated by this step is missing VLANs.

In main I get this:

```
[zuul@controller-0 architecture]$ kustomize build examples/va/hci/ | grep vlan
      "type": "macvlan",
      "type": "macvlan",
      "type": "macvlan",
      "type": "macvlan",
      vlan: 20
      vlan: 21
      vlan: 22
      vlan: 23
[zuul@controller-0 architecture]$ 
```

In the branch for this PR I get this:
```
[zuul@controller-0 architecture]$ kustomize build examples/va/hci/control-plane | grep vlan
      "type": "macvlan",
      "type": "macvlan",
      "type": "macvlan",
      "type": "macvlan",
[zuul@controller-0 architecture]$ 
```

Reply:
Hmm, not sure why that is.  Both branches use the same underlying `lib` stuff for the networking config generation.

Reply:
I misdiagnosed. It is not related to your patch.

I wasn't testing with the same values file for both of the above outputs. The source from ci-framework is what caused this and it should be fixed by this.

  https://github.com/openstack-k8s-operators/ci-framework/pull/1103/